docs/superpowers/specs/2026-05-15-accompaniment-quality-design.md

```
@@ -0,0 +1,417 @@
+- **Pop/Rock** section header, then presets
+- **Blues** section header, then presets
+- **Jazz** section header, then presets
+- ... etc for each category
+- Separator
+- **Suggested for [Scale Name]** section with dynamically generated presets
+
+## File Layout
+
+New and modified files:
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)